### PR TITLE
Adding book-dash-collaboration image.md

### PR DESCRIPTION
**Pull request by SaraVilla**
### Summary

Adding book-dash-collaboration image to book dash landing page

### List of changes proposed in this PR (pull-request)

- [x] Image already in image folder
- [x] Added code and link for figure
- [x] added alt-text
- [ ] Reviewed image to make sure all looks fine preview

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by SaraVilla** [comment]
Yeah, I just saw that the preview doesn't show anything :( 

I'll try again!

File: book/website/community-handbook/bookdash.md [hunk 1]
```diff
@@ -4,6 +4,15 @@
 _The Turing Way_ [Book Dash events](https://the-turing-way.netlify.app/community-handbook/bookdash.html) are a less intense version of [Book Sprints](https://en.wikipedia.org/wiki/Book_sprint), where participants collaboratively work on _The Turing Way_ book synchronously to develop new chapters and review/edit existing ones to make them more accessible, comprehensive and up-to-date.
 They also contribute to enhancing the project by improving the ways we work in the community and take the lead on accomplishing different tasks or subprojects.
 
+```{figure} ../../figures/book-dash-collaboration.*
```
Comment (EKaroune) [inline book/website/community-handbook/bookdash.md:7]:
```suggestion
```{figure} ../figures/book-dash-collaboration.*
```

**Comment by EKaroune** [comment]
I've changed the path to the image as sometimes it wants it to be slightly different - no idea why??

**Review by Arielle-Bennett — CHANGES_REQUESTED** [review]
Wasn't building correctly because you missed the file ending out I think - accept this change and see if that works!

File: book/website/community-handbook/bookdash.md [hunk 2]
```diff
@@ -4,6 +4,15 @@
 _The Turing Way_ [Book Dash events](https://the-turing-way.netlify.app/community-handbook/bookdash.html) are a less intense version of [Book Sprints](https://en.wikipedia.org/wiki/Book_sprint), where participants collaboratively work on _The Turing Way_ book synchronously to develop new chapters and review/edit existing ones to make them more accessible, comprehensive and up-to-date.
 They also contribute to enhancing the project by improving the ways we work in the community and take the lead on accomplishing different tasks or subprojects.
 
+```{figure} ../figures/book-dash-collaboration.*
+---
+height: 500px
+name: book-dash-collaboration
+alt: Cartoon-like sketch of a big yellow tree in the centre of image. A diverse group of people, all pictured within a yellow, green and orange colour scheme, including a person using a wheelchair and people from different backgorunds are watering and maintaining it with the help of 2 ladders. The title reads 'The Turing Way book dash.
```
Comment (EKaroune) [inline book/website/community-handbook/bookdash.md:11]:
```suggestion
alt: Cartoon-like sketch of a big yellow tree in the centre of image. A diverse group of people, all pictured within a yellow, green and orange colour scheme, including a person using a wheelchair and people from different backgrounds are watering and maintaining the tree with the help of 2 ladders. The title reads 'The Turing Way book dash.
```

**Review by EKaroune — APPROVED** [review]
looks good to me! :smile:

**Pull request merged**